dashboard/src/components/ConfirmDialog/index.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { mount } from "enzyme";
+import * as React from "react";
+import * as Modal from "react-modal";
+
+import ConfirmDialog from "./index";
+
+it("renders loading message", () => {
+  const wrapper = mount(
```

Comment:
I don't think you need to use `mount` here. Please use `shallow` where possible. See difference here: https://gist.github.com/fokusferit/e4558d384e4e9cab95d04e5f35d4f913